assets/src/edit-story/components/inspector/prepublish/components/checklistTab.js

```diff
@@ -243,81 +212,81 @@ const ChecklistTab = ({
     [renderRow]
   );
 
+  // Determine what to make visible based on checkpoint and present issues.
   const highPriorityLength =
     highPriority.length + (pages.lengths?.highPriority || 0);
+  const hasHighPriorityItems = Boolean(highPriorityLength);
+  const isHighPriorityVisible =
+    !isHighPriorityDisabledState && hasHighPriorityItems;
+
   const recommendedLength =
     recommended.length + (pages.lengths?.recommended || 0);
-  const hasHighPriorityItems = Boolean(highPriorityLength);
   const hasRecommendedItems =
-    Boolean(recommended.length) || Boolean(pages.lengths?.recommended);
+    Boolean(recommended.length) ||
+    Boolean(pages.lengths?.recommended) ||
+    (canOptimizeVideo && !areVideosAutoOptimized);
+  const isRecommendedVisible =
+    !isRecommendedDisabledState && hasRecommendedItems;
 
-  const isRecommendedDisabled =
-    isRecommendedDisabledState || !hasRecommendedItems;
-  const isHighPriorityDisabled =
-    isHighPriorityDisabledState || !hasHighPriorityItems;
+  const isEmptyView = useMemo(() => {
+    return (
+      currentCheckpoint === PPC_CHECKPOINT_STATE.UNAVAILABLE ||
+      isChecklistEmpty ||
+      (!isRecommendedVisible && !isHighPriorityVisible)
+    );
+  }, [
+    currentCheckpoint,
+    isRecommendedVisible,
+    isHighPriorityVisible,
+    isChecklistEmpty,
+  ]);
 
-  return isChecklistEmpty ? (
-    <EmptyChecklist />
-  ) : (
+  if (isEmptyView) {
+    return <EmptyChecklist body={TEXT.UNAVAILABLE_BODY} />;
```

Comment:
right now each empty condition has the same text, i figure it's unlikely to stay that same way so I'm passing the text into the component here so it's easier to update. we do the same in the dashboard. we could even pass in checkpoints too, like the icon set up for the publish button and checklist tab in the inspector panel, should there be updated assets.